TombEngine/Renderer/Renderer11Draw.cpp

```diff
@@ -2002,7 +2008,13 @@ namespace TEN::Renderer
 			BindConstantBufferVS(CB_ROOM, m_cbRoom.get());
 			BindConstantBufferPS(CB_ROOM, m_cbRoom.get());
 
-			SetScissor(room->Clip);
+			RendererRectangle rect;
```

Comment:
Can we maybe move this calculation to `RendererRoom` subroutine or something?

Reply:
It's part of my polish intentions like I have said about the scissor rect.